platform/nephos/rules.mk

```diff
@@ -14,6 +14,10 @@ include $(PLATFORM_PATH)/docker-ptf-nephos.mk
 NPX_DIAG = npx_diag
 $(NPX_DIAG)_URL = "https://github.com/NephosInc/SONiC/raw/master/sdk/npx_diag"
 
+WARM_VERIFIER = warm-verifier
+$(WARM_VERIFIER)_URL = "https://github.com/NephosInc/SONiC/raw/master/sai/warm-verifier"
```

Comment:
Both URL are pointing to the latest binary on master branch. It is difficult to track the version.

Please provide these information either in URL or in comments:
1. version
2. commit ID of the private code
3. MD5 of the binary
4. support email